src/block-kit/Modal.tsx

```diff
@@ -61,15 +65,6 @@ export const Modal: JSXSlack.FC<ModalProps> = props => {
     return undefined
   })()
 
-  const privateMetadata = (() => {
```

Comment:
The check of coverage in the whole of project by CodeCov was failed by reduced codes in this PR's change.